terragrunt/modules/docs-rs/builder.tf

```diff
@@ -32,7 +32,14 @@ resource "aws_iam_role_policy" "builder_s3" {
       // Access to s3
       {
         Effect = "Allow"
-        Action = "s3:*"
+        Action = [
+          "s3:PutObject",
+          "s3:GetObject",
+          "s3:CreateBucket",
+          "s3:ListBucket",
+          "s3:PutObjectTagging",
+          "s3:DeleteObject"
```

Comment:
we're actually only using the `DeleteObjects` action, my guess would be that's a separate permission?

Reply:
As far as I can tell, there is only `DeleteObject` which is used whether deleting a single object or many objects.